keras_cv/layers/vit_layers.py

```diff
@@ -87,9 +87,7 @@ def call(self, images):
         return patches
```

Comment:
can you validate self.padding in the constructor instead of in call()?

Reply:
Added validation in the latest push

Reply:
Done